hedera-node/src/main/java/com/hedera/services/stream/RecordStreamManager.java

```diff
@@ -286,6 +292,13 @@ public static String effLogDir(String baseDir, final String accountMemo) {
 		return baseDir + "record" + accountMemo;
 	}
 
+	public static String effSidecarLogDir(String baseRecordFileDir, final String sidecarDir) {
```

Comment:
Is this short for `effectiveSidecarLogDir`?
Could change to `getSideCarLogDir`

Reply:
Yup, was following the naming of `effLogDir` for the record file path, since it is not strictly a "getter" function, because additional logic is done inside the method to calculate the final path. I'm fine going either way, should we go ahead and change it?

Reply:
I prefer the more explicit once, names should provide some meaning and the abbreviated one doesn't at least to me

Reply:
Got it, changed both methods' names to be more explicit ---  `eff` -> `effective`